plugins/nextcloud-rule-exclusions-before.conf

```diff
@@ -845,36 +855,86 @@ SecRule REQUEST_FILENAME "@rx /index\.php/apps/notes/api/v[0-9]/notes" \
     pass,\
     t:none,\
     nolog,\
-    ver:'nextcloud-rule-exclusions-plugin/1.0.0',\
     ctl:ruleRemoveTargetByTag=OWASP_CRS;ARGS:json.content,\
-    ctl:ruleRemoveTargetByTag=OWASP_CRS;ARGS:json.title"
+    ctl:ruleRemoveTargetByTag=OWASP_CRS;ARGS:json.title,\
+    ver:'nextcloud-rule-exclusions-plugin/1.0.0'"
 
 # Note name and content can be anything
 # PUT - when making changes/creating notes
 # DELETE - when notes are deleted
 SecRule REQUEST_FILENAME "@contains /apps/notes/notes/" \
-    "id:9508920,\
+    "id:9508911,\
```

Comment:
Why did you change the ID?

Reply:
I changed it to make room for new rule exclusions, more of a cleanliness thing.

Reply:
I like that, in general. What I'm not sure of is how stable we need to keep the IDs of individual rules. @dune73, any thoughts?

Reply:
I've reverted the rule ID changes for now, this could be done in a separate PR